src/styleguide/_patterns/05-pages/00-page-layouts/00-basic-page.json

```diff
@@ -5,19 +5,13 @@
   },
   "links": [
     {
-      "url": false,
+      "url": "link.pages-about",
       "control": {
         "label": "About us"
       }
     },
     {
-      "url": "https://medium.com/buildit",
-      "control": {
-        "label": "Blog"
-      }
-    },
-    {
-      "url": "link.pages-careers",
+      "url": "link.careers",
```

Comment:
This was correct before: It should be `link.pages-careers`.

Reply:
Whoops, took a punt and got caught out by the watch process again!